### DISCUSSION TITLE
Welcome to TipBot Discussions!

### DISCUSSION THREAD

**Discussion by wrkzdev** [wrkzcoin/TipBot#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mystgryphs** [comment]
Hey do you guys respond?

**Reply by wrkzdev** [reply]
Hey there,

There are many ways to approach us to get response such as twitter, here or in our Discord: http://chat.wrkz.work

**Comment by mystgryphs** [comment]
Hello Pluton

I appreciate you replying and I would just love to ask.. is the tipbot
worth it? If i were to spend my time getting rewards. Can you guarantee
I'll be able to withdraw my rewards?

**Reply by wrkzdev** [reply]
Our TipBot has rewards with games and faucets and it's free for everyone to play. Please do not expect that there is big rewards from it. There's around 142k+ times of faucet claiming excluding text games and vote reward.

For withdraw, each coin or token has a minimum withdraw. We keep tipping various coins and tokens in our main Discord guild and if you have any issue withdrawing, you can ping me pluton#8888 in Discord and we are happy to assist.

**Comment by mystgryphs** [comment]
Yes i understand that there is a minimum requirement. I thought dego was 50
000 dego minimum requirement, but turns out that was the network fee and I
need 200 million dego in order to withdraw. Now my question to you is. If
all that time has passed and I can finally withdraw. What guarantee is
there that I'll get the money?

**Reply by wrkzdev** [reply]
You can check info by `/coininfo ...` or when you do with `/deposit ... `,

Example of mine:
![image](https://user-images.githubusercontent.com/40448869/204707983-fbba623e-343b-49b2-9bab-6e7951596881.png)

![image](https://user-images.githubusercontent.com/40448869/204708479-305629c8-3eeb-48b7-ad8f-eb0f7529d0ee.png)

I can do a withdraw on demand on if you request. Just ping me in our Discord. We limit them because it's easy to get a million of DeroGold by tipping from other people in Discord and reduced TipBot's load also less number of transactions..

> What guarantee is there that I'll get the money?

Our TipBot is holding only cryptocurrencies. Each withdraw is sending to each coin's network. Once it's confirmed in its network, it's there.

Here is explorer of DeroGold https://derogold4ever.online/

**Comment by mystgryphs** [comment]
So you're telling me you can withdraw my crypto for me? And if so how do I
get into the discord?
